src/Aquarius.Client/Aquarius.Client.csproj

```diff
@@ -26,10 +26,10 @@
   </ItemGroup>
 
   <!-- .NET Standard 2.1 references, compilation flags and build options -->
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' Or '$(TargetFramework)' == 'net6.0'">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' Or '$(TargetFramework)' == 'net8.0'">
     <DefineConstants>NODATIME2</DefineConstants>
   </PropertyGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' Or '$(TargetFramework)' == 'net6.0'">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' Or '$(TargetFramework)' == 'net8.0'">
     <PackageReference Include="NodaTime" Version="2.2.3" />
     <PackageReference Include="ServiceStack.Client.Core" Version="6.0.2" />
```

Comment:
We probably can't upgrade the Framework 4.7.2 versions of ServiceStack here, but shouldn't we upgrade the NET 8 versions to 8 so they will be consitent with WebPortal and Connect on .NET 8?

Reply:
I checked, and Servicestack v8 still supports 4.7.2